test/round-test.js

```diff
@@ -2,4 +2,152 @@ import chai from 'chai';
+    expect(round.returnCurrentPlayer()).to.equal(Player2);
+    round.incremenTurnCount()
+    expect(round.turnCount).to.equal(2);
+    expect(round.returnCurrentPlayer()).to.equal(Player3);
+    round.incremenTurnCount()
+    expect(round.turnCount).to.equal(0);
+    expect(round.returnCurrentPlayer()).to.equal(Player1);
+  })
+  it('should transfer winnings from score to bank for current player', function() {
+    round.returnCurrentPlayer()
+    round.incremenTurnCount();
+    expect(round.returnCurrentPlayer().score).to.equal(0);
+    expect(round.returnCurrentPlayer().bank).to.equal(0);
+    round.returnCurrentPlayer().score = 200;
+    expect(round.returnCurrentPlayer().score).to.equal(200);
+    round.roundOver();
+    expect(round.returnCurrentPlayer().score).to.equal(0);
+    expect(round.returnCurrentPlayer().bank).to.equal(200);
+  });
+  it('should evaluate the total word guess', function () {
+    Player1.score = 200;
+    Player2.score = 300;
+    Player3.score = 600;
+    round.guessAnswer('guess');
+    expect(Player1.score).to.equal(200);
+    expect(Player2.score).to.equal(300);
+    expect(Player3.score).to.equal(600);
+    expect(Player1.bank).to.equal(0);
+    expect(Player2.bank).to.equal(0);
+    expect(Player3.bank).to.equal(0);
+    round.guessAnswer('Armchair');
+    expect(Player1.score).to.equal(0);
+    expect(Player2.score).to.equal(0);
+    expect(Player3.score).to.equal(0);
+    expect(Player1.bank).to.equal(0);
+    expect(Player2.bank).to.equal(300);
+    expect(Player3.bank).to.equal(0);
+  })
+  it('should evaluate the players letter guess', function () {
+    Player1.score = 400;
+    Player2.score = 250;
+    Player3.score = 50;
+    round.wheel.selectCurrentValues();
+    let result = round.guessLetter('A');
+    // console.log('b', result);
+    if (result === 'BANKRUPT') {
+      expect(Player1.score).to.equal(0);
+    } else if (result === 'LOSE A TURN') {
+      expect(Player1.score).to.equal(400);
+      expect(round.returnCurrentPlayer()).to.eql(Player2);
+    } else if (result === 0) {
+      expect(Player1.score).to.equal(400);
+      expect(round.returnCurrentPlayer()).to.eql(Player2);
+    } else {
+      console.log(result);
+      expect(Player1.score).to.equal(result);      
+    }
+  })
+  it('should reject the players incorrect letter guess', function () {
+    Player1.score = 400;
+    Player2.score = 250;
+    Player3.score = 50;
+    round.wheel.selectCurrentValues();
+    let result = round.guessLetter('Z');
+    // console.log('b', result);
+    if (result === 'BANKRUPT') {
+      expect(Player1.score).to.equal(0);
+    } else if (result === 'LOSE A TURN') {
+      expect(Player1.score).to.equal(400);
+      expect(round.returnCurrentPlayer()).to.eql(Player2);
+    } else if (result === 0) {
+      expect(Player1.score).to.equal(400);
+      expect(round.returnCurrentPlayer()).to.eql(Player2);
+    } else {
+      expect(Player1.score).to.equal(result);      
```

Comment:
Let's break this conditional apart so that we have separate testing blocks (we shouldn't have conditionals, loops, etc in our tests)